### PR TITLE
Include OCR data in queue messages

### PR DESCRIPTION
**Pull request by lgonczar**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/BPS-253

### Change description ###

Include OCR data, if present in the input metadata, in messages for the orchestrator.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Comment by lgonczar** [comment]
We can't add a functional test verifying the queue message, as it's not guaranteed that the message won't be consumed by the orchestrator first. As for unit and integration tests, OCR data was partly handled (it already existed in the envelope). Other than that, this field alone didn't create any edge cases, so I only had to add an assertion to a test or value to test data.

File: src/test/resources/metafiles/valid/from-spec.json [hunk 1]
```diff
@@ -15,6 +15,7 @@
       "next_action": "return",
       "next_action_date": "24-02-2017 00:00:00.000010",
       "file_name": "1111001.pdf",
+      "ocr_data": "VGVzdCBvY3IgZGF0YQo=",
```
Comment (rkondratowicz) [inline src/test/resources/metafiles/valid/from-spec.json]: do we know whether this will be a base64 encoded json?

Comment (doncem) [reply src/test/resources/metafiles/valid/from-spec.json]: yes, we [do](https://tools.hmcts.net/confluence/display/RBS/Technical+Specification+-+Exela#TechnicalSpecification-Exela-3.4Metafilefields)

**Comment by lgonczar** [comment]
As discussed, the processor will parse OCR data and send it as a String->String map. Closing.
